### ISSUE TITLE
Fail with a scala project

### ISSUE THREAD

**Issue by dacr** [STAMP-project/pitest-descartes#101]
#### Characteristics
- **Issue Type**: feature
- **Reproducibility**: always
- **Severity**: crash
- **Tool/Service/Component**: [descartes, 1.2.5]
- **Execution Environment**: [scala]
- **Reporter**: [crosson david]

#### Description
Got thisissue while trying to use descarte on a scala project : 
```
15:01:11 PIT >> INFO : Verbose logging is disabled. If you encounter a problem, please enable it before reporting an issue.
15:01:14 PIT >> INFO : Sending 411 test classes to minion
15:01:14 PIT >> INFO : Sent tests to minion
15:01:14 PIT >> INFO : MINION : 15:01:14 PIT >> INFO : Checking environment

15:01:16 PIT >> INFO : MINION : 15:01:16 PIT >> INFO : Found  1 tests

15:01:16 PIT >> INFO : MINION : 15:01:16 PIT >> INFO : Dependency analysis reduced number of potential tests by 0

15:01:16 PIT >> INFO : MINION : 15:01:16 PIT >> INFO : 1 tests received

/15:01:17 PIT >> INFO : Calculated coverage in 4 seconds.
Exception in thread "main" java.util.NoSuchElementException: Attempt to grab a line from an empty interval in: com.orange.xperf.parsers.config.apache.MinSpareServersDirective at unapply with (Lcom/orange/xperf/parsers/config/apache/MinSpareServersDirective;)Lscala/Option;
	at eu.stamp_project.mutationtest.descartes.bodyanalysis.LineCounter.ensureNotEmpty(LineCounter.java:51)
	at eu.stamp_project.mutationtest.descartes.bodyanalysis.LineCounter.getFirstLine(LineCounter.java:40)
	at eu.stamp_project.mutationtest.descartes.bodyanalysis.MethodInspector.visitEnd(MethodInspector.java:28)
	at org.pitest.reloc.asm.ClassReader.readMethod(ClassReader.java:1287)
	at org.pitest.reloc.asm.ClassReader.accept(ClassReader.java:688)
	at org.pitest.reloc.asm.ClassReader.accept(ClassReader.java:400)
	at eu.stamp_project.mutationtest.descartes.DescartesMutater.getMutationPoints(DescartesMutater.java:40)
	at eu.stamp_project.mutationtest.descartes.DescartesMutater.findMutations(DescartesMutater.java:34)
	at org.pitest.mutationtest.build.MutationSource.createMutations(MutationSource.java:55)
	at org.pitest.mutationtest.build.MutationTestBuilder.lambda$classToMutations$1(MutationTestBuilder.java:94)
	at org.pitest.functional.FCollection.flatMapTo(FCollection.java:58)
	at org.pitest.functional.FCollection.flatMap(FCollection.java:68)
	at org.pitest.mutationtest.build.MutationTestBuilder.createMutationTestUnits(MutationTestBuilder.java:58)
	at org.pitest.mutationtest.tooling.MutationCoverage.buildMutationTests(MutationCoverage.java:281)
	at org.pitest.mutationtest.tooling.MutationCoverage.runReport(MutationCoverage.java:135)
	at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:120)
	at org.pitest.mutationtest.tooling.EntryPoint.execute(EntryPoint.java:50)
	at org.pitest.mutationtest.commandline.MutationCoverageReport.runReport(MutationCoverageReport.java:87)
	at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:45)
	at TmpMain$.main(TmpMain.scala:12)
	at TmpMain.main(TmpMain.scala)
```

#### [Help on issue template](https://github.com/STAMP-project/pitest-descartes/blob/master/.github/ISSUE_DOC.md) 
Preview to follow the link or open file .github/ISSUE_DOC.md

**Comment by oscarlvp** [comment]
@dacr Thanks for reporting. This issue should have been solved with commit 2f46d26. You can try the code that is currently in the master branch and build descartes locally. Use then version 1.2.6-SNAPSHOT.

**Comment by oscarlvp** [comment]
Version 1.2.6-SNAPSHOT works with other Scala projects.
